clusters/app.ci/supplemental-ci-images/golangci-lint_is.yaml

```diff
@@ -15,3 +15,7 @@ spec:
     from:
       kind: DockerImage
       name: "docker.io/golangci/golangci-lint:v1.37.1"
+  - name: "v1.43.0"
+    from:
+      kind: DockerImage
+      name: "docker.io/golangci/golangci-lint:v1.43.0"
```

Comment:
Not here, please follow https://docs.ci.openshift.org/docs/how-tos/external-images/

Reply:
Ack, will do it, thanks.

I guess that I need someone from the CI team to approve the PR?

Reply:
Correct